images/docker/cbdb/build/rocky8/Dockerfile

```diff
@@ -0,0 +1,196 @@
+# --------------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed
+# with this work for additional information regarding copyright
+# ownership. The ASF licenses this file to You under the Apache
+# License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the
+# License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+# --------------------------------------------------------------------
+#
+# Apache Cloudberry (incubating) is an effort undergoing incubation at
+# the Apache Software Foundation (ASF), sponsored by the Apache
+# Incubator PMC.
+#
+# Incubation is required of all newly accepted projects until a
+# further review indicates that the infrastructure, communications,
+# and decision making process have stabilized in a manner consistent
+# with other successful ASF projects.
+#
+# While incubation status is not necessarily a reflection of the
+# completeness or stability of the code, it does indicate that the
+# project has yet to be fully endorsed by the ASF.
+#
+# --------------------------------------------------------------------
+# Dockerfile for Apache Cloudberry Build Environment
+# --------------------------------------------------------------------
+# This Dockerfile sets up a Rocky Linux 9-based container for building
```

Comment:
Your contributions to the team have been invaluable. I just finished reading all the code. The process is very smooth and easy to understand. This is very helpful for me to understand the entire process. There is one place where I am wondering if it was a mistake. This Dockfile should be established as a Rocky Linux 8-based container for building.

Reply:
@Zhangbaowen-Hashdata Thank you for the thorough review and kind feedback. I deliberately chose Rocky Linux 9 as our initial base platform. The plan is to expand support to additional platforms (including Rocky Linux 8, Ubuntu, and Debian) in upcoming iterations. For now, I wanted to focus on establishing solid automation processes for Apache Cloudberry (incubating) using a single platform as our foundation. Once we have that working smoothly, expanding to other distributions will be more straightforward.